### PR TITLE
Dp 12320 create callout alert

### PR DESCRIPTION
**Pull request by clairesunstudio**
screen shot 2019-01-23 at 12 12 07 pm

Brought in Patternlab Callout Alert Pattern and added themes and icon options.
This pr also added the alert icon and simplified icon options to be used in other stories

### CODE REVIEW

File: react/src/components/organisms/CalloutAlert/CalloutAlert.stories.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  'c-gray-dark': 'c-gray-dark',
+  'c-error-red': 'c-error-red'
+};
+
+storiesOf('organisms/CalloutAlert', module).addDecorator(withKnobs)
+  .add('CalloutAlert', withInfo(`<div>${CalloutAlertDocs}</div>`)(() => {
+    const name = select('CalloutAlert.icon', svgOptions, '');
+    const theme = select('CalloutAlert.theme', themeOptions, '');
+    const calloutAlertProps = {
+      icon: { name },
```
Comment (avrilpearl) [inline react/src/components/organisms/CalloutAlert/CalloutAlert.stories.js]: do you want to pass the title here as well in the story?

**Review by avrilpearl — APPROVED** [review]

File: react/src/components/organisms/CalloutAlert/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        {props.children}
+      </div>
+    </div>
+  );
+};
+
+CalloutAlert.propTypes = {
+  icon: PropTypes.shape(Icon.propTypes),
+  /** Themes correspond to site color scheme i.e. sass variables */
+  theme: PropTypes.oneOf(['', 'c-primary-alt', 'c-highlight', 'c-gray-dark', 'c-error-red']),
```
Comment (smurrayatwork) [inline react/src/components/organisms/CalloutAlert/index.js]: This is missing the theme c-primary:
```suggestion
  theme: PropTypes.oneOf(['', 'c-primary', 'c-primary-alt', 'c-highlight', 'c-gray-dark', 'c-error-red']),
```

File: react/src/components/organisms/CalloutAlert/index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  children: PropTypes.element
```
Comment (smurrayatwork) [inline react/src/components/organisms/CalloutAlert/index.js]: If more than one child is passed, this is actually an array:
```suggestion
  children: PropTypes.oneOfType([PropTypes.element, PropTypes.array])
```

**Review by smurrayatwork — CHANGES_REQUESTED** [review]
Just need to apply the two suggestions and then this is good to go.

**Review by smurrayatwork — APPROVED** [review]

**Pull request merged**